schemas/context/profileStitch.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profileStitch",
```

Comment:
Also needs `@id`, as per new plans

Reply:
Agreed, we need to get that approved first and then we will do the needful. I would like Kevin to return for the `@id` decision and exactly how we roll it out. In the XC we cannot automatically add them, so we need to do something to limit the explosion of fields in our operational store.